install_config/configuring_routing.adoc

```diff
@@ -10,25 +10,44 @@
 
 toc::[]
 
-
 == Overview
-This topic describes how to set up container networking using existing switches
-and routers and the kernel networking stack in Linux. The setup requires that
-the network administrator or a script modifies the router or routers when new
-nodes are added to the cluster.
+After installing {product-title} and creating an existing router, you can modify
+it to suit your needs using the examples in this topic.
 
-[NOTE]
-The procedures outlined in this topic can be adapted to any type of router.
+== Configuring Router Timeouts
+You can configure the default timeouts for an existing route when you have
+services in need of a low timeout, as required for Service Level Availability
+(SLA) purposes, or a high timeout, for cases with a slow back end.
 
+Using the `oc annotate` command, add the timeout to the route:
 
-== Network Layout
-The following diagram shows the container networking setup described in this
-topic. It uses one Linux node with two network interface cards serving as a
-router, two switches, and three nodes connected to these switches.
+====
+----
+# oc annotate route <route_name> \
+    --overwrite haproxy.router.openshift.io/timeout=<timeout>s
```

Comment:
The timeout does not have to be in seconds, it can be in any unit that haproxy supports. By default (if no unit is given) the timeout is in milliseconds but can be microseconds (us), milliseconds (ms), seconds (s), minutes (m), hours (h), or days (d).